decompiler/config/jak2/all-types.gc

```diff
@@ -29690,21 +29693,36 @@
 ;; enemy-h                        ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-#|
-(deftype enemy-focus (focus)
+;;Added hopefully temp structure since "focus" used the 12 method slot instead of enemy-focus
```

Comment:
It should be possible to make this work without needing a separate type like this.

The type of method 12 in focus and enemy-focus should be the same (though it's possible that I got it wrong in `focus` and you'll need to add an additional argument).

I think it's probably going to be something like:
```
(_type_ process-focusable enemy)
```